src/server/repository/assessmentCycle/repository/getOne.ts

```diff
@@ -8,14 +9,20 @@ import { BaseProtocol, DB, Schemas } from 'server/db'
 type Props = {
   assessment: Assessment
   cycle: Cycle
+  countryIso?: CountryIso
 } & ({ uuid: string } | { fileName: string })
 
 export const getOne = async (props: Props, client: BaseProtocol = DB): Promise<RepositoryItem> => {
-  const { assessment, cycle } = props
+  const { assessment, countryIso, cycle } = props
   const schemaCycle = Schemas.getNameCycle(assessment, cycle)
 
   const values = 'uuid' in props ? [props.uuid] : [props.fileName]
-  const where = 'uuid' in props ? 'uuid = $1' : "props -> 'translation' ->> 'en' = $1"
+  let where = 'uuid' in props ? 'uuid = $1' : "props -> 'translation' ->> 'en' = $1"
+
+  if (!Objects.isEmpty(countryIso)) {
```

Comment:
It should be unique, yes, I will remove the param. But good to note, there isn't any database level unique check so two files can't be in the same country with the same name. 

One potential issue we could have is if a user adds a pdf file to the country repository with the same name as the one we generate for the reports (e.g `fra_2020_CAF.pdf`), then the `client.one` function in the repository will throw an error (multiple rows match), and generating a report would fail. 
Should we add any logic to handle this? Since the file name is all we have to query an existing report, a solution I see is adding a new param to the repository props like 'report: true', and then we could also use that to make the query more specific.